modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -732,8 +732,9 @@ void cv::gimpl::ie::GIEExecutable::run(cv::gimpl::GIslandExecutable::IInput  &in
 
     if (cv::util::holds_alternative<cv::gimpl::EndOfStream>(in_msg))
     {
-        // (3) Wait until all passed task are done.
+        GAPI_LOG_INFO(nullptr, "Got EndOfStream - wait until all passed task are done");
```

Comment:
Troubleshooting  & Logging BKMs:

INFO level in log here because pipeline is intending to stop: it's once-time event, not need to worry about performance due to printout here